docs/getting-started-step-by-step/00-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview Getting Started: Step-by-Step
```

Comment:
Is the intention that this replaces or supplements the existing getting started guide? It seems like there is quite a lot of duplication with existing sections (e.g. how to install Crossplane).

Reply:
The goal is to supplement it. I've clarified it as a goal in the description. And added a task: "Integrate it into the current getting started"